plasmapy/formulary/mathematics.py

```diff
@@ -92,3 +92,43 @@ def Fermi_integral(
         return integral_arr
     else:
         raise TypeError(f"Improper type {type(x)} given for argument x.")
+
+
+def rot_a_to_b(a, b):
+    r"""
+    Calculates the 3D rotation matrix that will rotate vector a to be aligned
+    with vector b.
+
+    Parameters
+    ----------
+    a : ~np.ndarray, shape (3)
+        Pre-rotation orientation unit vector
+
+    b : ~np.ndarray, shape (3)
+        Post-rotation orientation unit vector
+
+    Returns
+    -------
+    R : ~np.ndarray, shape(3,3)
+        The rotation matrix that will rotate vector a to be parallel to
+        vector b.
+
+
+    The algorithm is based on `this discussion <https://math.stackexchange.com/questions/180418/calculate-rotation-matrix-to-align-vector-a-to-vector-b-in-3d/476311#476311>` on StackExchange:
```

Comment:
```suggestion
    Notes
    -----
    The algorithm is based on
    `this discussion <https://math.stackexchange.com/a/476311>`_ on StackExchange:
```

Note, for linking in `sphinx` you need to have the trailing underscore `` `label <target>`_ ``.

I think you should also include the associated math in this "Notes" section.